app/page.tsx

```diff
@@ -136,6 +137,18 @@ export default function Home() {
           />
           Repo
         </a>
+        <Link
+          className="hover:underline hover:underline-offset-4"
+          href="/how-to-work-with-me"
+        >
+          How to Work With Me
+        </Link>
+        <Link
+          className="hover:underline hover:underline-offset-4"
+          href="/how-i-handle-your-data"
+        >
+          How I Handle Your Data
+        </Link>
```

Comment:
Let's add svg icons to each of these two similar to the Globe on Repo. I don't really care what they are